packages/gatsby-theme-docs/src/utils/release-notes.js

```diff
@@ -15,7 +28,7 @@ const mapSiteTitleToFacetFilter = new Map([
   ['Import API', { group: 'product', productArea: 'Import API' }],
   [
     'Merchant Center Customizations',
-    { productArea: 'Merchant Center Customizations' },
+    { group: 'product', productArea: 'Merchant Center Customizations' },
```

Comment:
If we implement it like this, it would mean that we have to add all new microsites that we want to create in the docs repo here as well...which means that we would need to go through a kit-release as part of creating a new docs microsite.
If possible, I would like to avoid that...